### DISCUSSION TITLE
New 'clr' icon type is incomplete

### DISCUSSION THREAD

**Discussion by jfbauer432** [weather-gov/api#333] [Q&A] [answered]
Noticed today a new icon type of 'clr' appearing in a station current conditions.  My guess is that is has the same meaning as 'skc', but there are a couple of problems with it.

https://api.weather.gov/stations/KGAI/observations/latest
has "properties"."icon" of "https://api.weather.gov/icons/land/night/clr?size=medium".  However,

1. There is no 'clr' listed in https://api.weather.gov/icons
2. https://api.weather.gov/icons/land/night/clr does not return an icon (you get a status 500 in json)

BTW, are there any other new icon names?

x-correlation-id 8a754c88-87b3-451e-9a46-af22b25dd712

**Comment by StephenClouse** [comment, marked as answer]
Treat it as "skc".

We'll have a hotfix to correct this later this week.
